features/environment.py

```diff
@@ -128,13 +129,18 @@ def _make_patroni_test_config(self, name, dcs, tags):
             config['tags'] = tags
 
         if dcs != 'etcd':
-            etcd = config.pop('etcd')
-            config['zookeeper'] = {'scope': etcd['scope'], 'session_timeout': etcd['ttl'],
-                                   'reconnect_timeout': config['loop_wait']}
-            if dcs == 'exhibitor':
-                config['zookeeper']['exhibitor'] = {'hosts': ['127.0.0.1'], 'port': 8181}
+            dcs_config = config.pop('etcd')
```

Comment:
I don't agree, In sample config files should be only one DCS section available. Otherwise it's really confusing.